examples/lwjgl/script/build.sh

```diff
@@ -1,10 +1,17 @@
-#!/bin/zsh -euo pipefail
+#!/usr/bin/env -S /bin/zsh -euo pipefail
```

Comment:
Why do you need env here?

Reply:
https://stackoverflow.com/questions/53582831/zsh-on-linux-doesnt-recognize-valid-options-in-a-shell-script